### PR TITLE
Partially prettier

### PR DESCRIPTION
**Pull request by fregante**
Prettier sometimes is good. I ran this and dropped any unwanted changes:

```sh
npx prettier . --write --single-quote --use-tabs --no-bracket-spacing --arrow-parens avoid
npx sort-package-json
```

### CODE REVIEW

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -3,54 +3,53 @@ name: Deployment
...
+      - run: npm install
+      - run: npm test # This includes the build
+      - uses: fregante/daily-version-action@v1
+        name: Create tag if necessary
+        id: daily-version
+      - name: Update extension’s meta
+        env:
+          VER: ${{ steps.daily-version.outputs.version }}
+        run: |
+          echo https://github.com/$GITHUB_REPOSITORY/tree/$VER > distribution/SOURCE_URL.txt
```
Comment (fregante) [inline .github/workflows/deployment.yml:27]: I added the extension here as well

File: readme.md [hunk 2]
```diff
@@ -39,100 +39,105 @@ The template bakes in a pretty basic webpack config, with no transpiling. To set
 
 1. Install Babel packages and respective loader for webpack.
 
-	``` sh
```
Comment (fregante) [inline readme.md:42]: It looks like the extra indentation was not necessary, so prettier dropped it. The output is unchanged: https://github.com/notlmn/browser-extension-template/blob/163a5b91683a7897c980629450994ba3248db77e/readme.md

Comment (fregante) [reply readme.md:42]: Well, actually it looks like it replaced the initial tab with 3 spaces… I'm reverting this part

File: package.json [hunk 3]
```diff
@@ -1,17 +1,35 @@
 {
 	"private": true,
 	"scripts": {
+		"build": "webpack --mode=production",
```
Comment (fregante) [inline package.json:4]: Changes to this file are due to https://github.com/keithamus/sort-package-json

File: readme.md [hunk 4]
```diff
@@ -181,4 +188,3 @@ Extension icon made by [Freepik](https://www.freepik.com) from [www.flaticon.com
 ## License
 
 [![CC0](https://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
-
```
Comment (notlmn) [inline readme.md:184]: Is an empty line removed here?

**Review by notlmn — APPROVED** [review]
Apart from the single comment, everything else looks good to me! Also a lot cleaner.

File: readme.md (hunk 4 above)
Comment (fregante) [reply readme.md:184]: Yes, likely because the file ended with `\n\n` instead of simply `\n`

**Pull request merged**